### DISCUSSION TITLE
token('sizes.foo') on flexBasis props results on no style

### DISCUSSION THREAD

**Discussion by azoriusss** [chakra-ui/panda#2684] [Q&A] [answered]
Hi everyone

I have a specific problem wich I assume is due to the fact that `flexBasis` property is waiting for `spacing` tokens (and no `sizes` token)

- `css({flexBasis:token('sizes.xl')})` generates className but no style ❌
- `css({flexBasis:'36em'})` generates both className and style ✅
- Using a workaround with style props & css var is working as well ✅

Is this a bug or a choice ? And should `flexBasis` accept `sizes` token too ?

Here is a [playground](https://play.panda-css.com/OvJmcpN39V)

Thank you for your answers 🙏

**Comment by anubra266** [comment, marked as answer]
@azoriusss The token method is for dynamic use; e.g. in jsx `style` attribute. It can't be statically extracted by panda.
Do this instead: https://play.panda-css.com/OvJmcpN39V/jZHjkfA2nI

**Reply by azoriusss** [reply]
Thank you !
